Library/Homebrew/requirements/arch_requirement.rb

```diff
@@ -14,21 +14,46 @@ class ArchRequirement < Requirement
   attr_reader :arch
 
   def initialize(tags)
-    @arch = tags.shift
+    @arch = []
+    tags.each do |tag|
+      if Hardware::CPU::ALL_ARCHS.include? tag
+        @arch.append(tag)
```

Comment:
Would it be easier to make an `@archs` array? Seems like it might simplify some logic below.

Reply:
Sure, I don't have objections, but just to clarify - does that mean we'd support both `:arch` and `:archs` symbols then? Do I need to add some validation logic to prevent both being declared at the same time? Or do we allow both?

Just thinking about the rollout strategy and existing formulae with `depends_on :arch` and how do we avoid breaking them?

Reply:
From a quick search I'm not sure that `attr_reader :arch` is actually ever used so I'd propose that be removed.

I propose only `:arch` be used.



> Just thinking about the rollout strategy and existing formulae with `depends_on :arch` and how do we avoid breaking them?

Ensure that multiple and single values both still work.